common/src/test/java/org/esbtools/message/admin/common/MetadataServiceTest.java

```diff
@@ -100,40 +113,26 @@ public void testUpdatingFields() {
         Assert.assertNull(child);
         child = fetchMetadataField("EntityPostUpdate", "entityPostUpdate", MetadataType.Entity);
         assertfieldAssertions("EntityPostUpdate", "entityPostUpdate", MetadataType.Entity, child);
-
-        // todo illegal type
     }
 
     @Test
-    public void testFieldDeletion() {
+    public void testMetadataFieldDeletion() {
         long id = -1L;
         MetadataResponse result = service.addChildMetadataField(id, "Entities", MetadataType.Entities, "entitiesDeleteTest");
         assertSuccess(result);
         MetadataField parent = fetchMetadataField("Entities", "entitiesDeleteTest", MetadataType.Entities);
-        result = service.addChildMetadataField(parent.getId(), "EntityDelete1", MetadataType.Entity, "entityDelete1");
-        assertSuccess(result);
-        result = service.addChildMetadataField(parent.getId(), "EntityDelete2", MetadataType.Entity, "entityDelete2");
+        result = service.addChildMetadataField(parent.getId(), "EntityDelete", MetadataType.Entity, "entityDelete");
         assertSuccess(result);
-        MetadataField child1 = fetchMetadataField("EntityDelete1", "entityDelete1", MetadataType.Entity);
-        assertfieldAssertions("EntityDelete1", "entityDelete1", MetadataType.Entity, child1);
-        MetadataField child2 = fetchMetadataField("EntityDelete2", "entityDelete2", MetadataType.Entity);
-        assertfieldAssertions("EntityDelete2", "entityDelete2", MetadataType.Entity, child2);
-        result = service.deleteMetadataField(child1.getId());
+        MetadataField child = fetchMetadataField("EntityDelete", "entityDelete", MetadataType.Entity);
+        assertfieldAssertions("EntityDelete", "entityDelete", MetadataType.Entity, child);
+        result = service.deleteMetadataField(child.getId());
         assertSuccess(result);
         Assert.assertEquals(result.getResult().getId().longValue(), parent.getId().longValue());
-        Assert.assertNull("Child1 was not deleted!", fetchMetadataField("EntityDelete1", "entityDelete1", MetadataType.Entities));
-
-        // result = service.deleteMetadataField(parent.getId());
-        // Assert.assertNull("Entities was not deleted!",
-        // fetchMetadataField("Entities", "entitiesDeleteTest",
-        // MetadataType.Entities));
-        // ensure child2 was not deleted to preserve history
-        child2 = fetchMetadataField("EntityDelete2", "entityDelete2", MetadataType.Entity);
-        assertfieldAssertions("EntityDelete2", "entityDelete2", MetadataType.Entity, child2);
+        Assert.assertNull("Child was not deleted!", fetchMetadataField("EntityDelete1", "entityDelete1", MetadataType.Entities));
     }
 
     @Test
-    public void resultCreationTest() {
+    public void resultCreationOnCrudOperationsTest() {
```

Comment:
Very minor nitpick here, the rest of the tests are prefixed with "test", and this one has it as a suffix